templates/inject_data.html

```diff
@@ -23,14 +23,25 @@
 
             <div id="output-section" class="code-section">
                 <h3 class="title">Wide Render</h3>
-                <iframe class="rendered-output-code-context" id="rendered-output-code-context-large"></iframe>
+                <div class="rendered-output-code-context" id="rendered-output-code-context-large"></div>
                 <h3 class="title">Narrow Render</h3>
-                <iframe class="rendered-output-code-context" id="rendered-output-code-context-small"></iframe>
+                <div class="rendered-output-code-context" id="rendered-output-code-context-small"></div>
                 <div id="title">Output Code</div>
                 <div id="contents">
                     <textarea id="output-code-area" rows="20" cols="100"></textarea>
                 </div>
             </div>
         </div>
     </body>
+    <script>
+    function hideDetails(z) {
```

Comment:
@CalebNeelCO, please use descriptive names. `id` would be fine for this in my opinion. Downside to single letter variable names: they're cryptic. Upside to descriptive names: they communicate programmer intent. Please also fix `x`.